app/build.gradle

```diff
@@ -33,38 +36,33 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile "com.android.support:design:$supportLibraryVersion"
-    compile 'com.android.support:multidex:1.0.1'
-
-    compile project(path: ':auth')
-    compile project(path: ':database')
-    compile project(path: ':firestore')
-    compile project(path: ':storage')
-
-    compile "com.google.android.gms:play-services-auth:$firebaseVersion"
-    compile "com.google.firebase:firebase-auth:$firebaseVersion"
-    compile "com.google.firebase:firebase-database:$firebaseVersion"
-    compile "com.google.firebase:firebase-storage:$firebaseVersion"
+    implementation "com.google.firebase:firebase-core:$firebaseVersion"
+    implementation "com.android.support:design:$supportLibraryVersion"
+    implementation 'com.android.support:multidex:1.0.2'
 
-    compile('com.facebook.android:facebook-login:4.27.0')
-    compile("com.twitter.sdk.android:twitter-core:3.0.0@aar") { transitive = true }
+    implementation project(path: ':auth')
+    implementation project(path: ':firestore')
+    implementation project(path: ':database')
+    implementation project(path: ':storage')
 
-    compile "android.arch.lifecycle:runtime:$architectureVersion"
-    compile "android.arch.lifecycle:extensions:$architectureVersion"
+    implementation 'com.facebook.android:facebook-login:4.27.0'
+    // Needed to override Facebook
+    implementation "com.android.support:cardview-v7:$supportLibraryVersion"
+    implementation "com.android.support:customtabs:$supportLibraryVersion"
```

Comment:
I couldn't figure how to override these on our side so devs will have to do this themselves.

Reply:
I believe this is actually the preferable action to take.